examples/options-api/basic/pages/index.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="container">
+    <button @click="increment">
+      Increment
+    </button>
+    <p>Counter : {{ counter }}</p>
+    <button @click="decrement">
+      Decrement
+    </button>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend({
+  data () {
+    return {
+      counter: 0
+    }
+  },
+  methods: {
+    increment () {
+      return this.counter++
+    },
+    decrement () {
+      return this.counter--
+    }
```

Comment:
These functions shouldn't be returning any value.
I suggest also always provide return type of functions in TypeScript projects (even if `void`)
```suggestion
    increment (): void {
      this.counter++
    },
    decrement (): void {
      this.counter--
    }
```